odoo/addons/test_read_group/tests/test_fill_temporal.py

```diff
@@ -0,0 +1,746 @@
+# -*- coding: utf-8 -*-
+from odoo.tests import common
+
+import functools
+import traceback
+
+class TestFillTemporal(common.TransactionCase):
+    """Test for fill temporal.
+
+    This feature is mainly used in graph view. For more informations, read the
+    documentation of models's '_read_group_fill_temporal' method.
+    """
+
+    def setUp(self):
+        super(TestFillTemporal, self).setUp()
+        self.Model = self.env['test_read_group.fill_temporal']
+
+    def test_date_range_and_flag(self):
+        """Simple date range test, the flag is also tested.
+
+        One of the most simple test. It must verify that dates 'holes' are filled
+        only when the fill_temporal flag is set.
+        """
+        self.Model.create({'date': '1916-08-18', 'value': 2})
+        self.Model.create({'date': '1916-10-19', 'value': 3})
+        self.Model.create({'date': '1916-12-19', 'value': 5})
+
+        groups = self.Model.with_context(fill_temporal=False).read_group([],
```

Comment:
By default the context is empty. Call `read_group` with the default context (don't add an explicit `fill_temporal=False`) for this case.